source/code/source/playfab/PlayFabAndroidHttpPlugin.cpp

```diff
@@ -251,6 +251,11 @@ namespace PlayFab
         if (container != nullptr && container->HandleInvalidSettings())
         {
             std::shared_ptr<RequestTask> requestTask = nullptr;
+            { // LOCK httpRequestMutex
+                std::unique_lock<std::mutex> lock(httpRequestMutex);
+                container->requestId = pendingRequests.size()+1;
```

Comment:
This is not the correct requestId.
The RequestId is extracted from the http response headers from the PlayFab API call.
It's information that the customer sends us when they experience a bug that they can't fix themselves, so we can look up their call in our internal logs.
It's not just an arbitrary value.